wagtail/admin/views/home.py

```diff
@@ -144,3 +146,12 @@ def default(request):
     redirected to the login page.
     """
     raise Http404
+
+
+@cache_page(60 * 60 * 24 * 365)  # seconds * minutes * hours * days
+def sprite(request):
+    response = render(request, "wagtailadmin/icons/sprite.svg")
+    response["Content-Type"] = "image/svg+xml"
+    if not settings.DEBUG:
+        response["Cache-Control"] = f"max-age={60 * 60 * 1}"  # seconds * minutes * hours
```

Comment:
Normal assets are under webserver cache policy (if configured). But this is a Django view, default without browser cache.

I experimented with infinite browser cache and invalidating the sprite with a hash in the filename. I think that will lead to many problems. Especially with multi worker setups. Each instance need to use the same hash. Keeping that in sync is a pain.

We need to update the filename `sprite-[hash].svg` as `#` is used to reference the symbols by id.

The sprite is only 80 kilobytes. But the whole idea of a sprite is to stop bytes to go over the wire each request.

So I choose one hour browser cache in production mode. Is that sensible?